### DISCUSSION TITLE
Use DLT Daemon to extract a file

### DISCUSSION THREAD

**Discussion by 8vp** [COVESA/dlt-daemon#677] [Q&A]
Hi all, is it possible to use any functionality built within DLT Daemon to pull a file (non DLT) that is stored in the file system. I am aware of the FileTransfer plug-in but I do not want to use it because of the message injection callback needed. 

The context is that we are having a set of non-textual log files created and stored in various ECUs. They all also run DLT Daemons. Now we want to collect all these log files into a central ECU and wondering if there is any utility provided by the DLT Deamon that can achieve this. Many thanks!

**Comment by lti9hc** [comment]
Hi @8vp,
You are right, as I know the current tool support for pulling a file is FileTransfer plug-in
